server/plugin/api_test.go

```diff
@@ -662,6 +662,11 @@ func TestHandleCreateSubscriptions(t *testing.T) {
 				FirstName: "mockCreatedBy",
 			}, nil)
 
+			privacySettings := model.PrivacySettings{}
```

Comment:
maybe something like this ?
```suggestion
			privacySettings := model.PrivacySettings{
			ShowFullName = &showFullName
			}
```